Source/Diagnostics/FullDiagnostics.H

```diff
@@ -25,6 +25,8 @@ private:
      * before writing the diagnostic.
      */
     bool m_solver_deposits_current = true;
+    /** Whether to apply k-space filtering of the charge density */
+    bool m_apply_filter_rho = false;
```

Comment:
I don't think that storing this attribute is needed. (In practice, we always use `true` for the `FullDiagnostic`, as shown below in the `.cpp` file.) 
```suggestion
```

Reply:
oops, it was a leftover from my previous draft implementation. Thank you for spotting this! Fixed in 14dd38d705.